cmd-graph.c

```diff
@@ -107,6 +107,7 @@ static void print_addr(struct field_data *fd)
 static struct display_field field_total_time= {
 	.id      = GRAPH_F_TOTAL_TIME,
 	.name    = "total-time",
+	.alias   = { "total", "time", NULL },
```

Comment:
Hmm.. this syntax seems to be a GCC extension thus not portable.  I couldn't find any other way initializing a flexible array member though.

Reply:
What about using only `total` instead of providing multiple aliases?

Reply:
OK, that would make it a lot easier. :)

Reply:
If you agree, I will modify and update this PR again :)

Reply:
It'd be nice if you could update it.

Reply:
Updated it. Thanks!